tests/ffi/qm_nested/test.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+# Declare global variables and initialize to 0
+NESTED_NAME=0
+IS_NESTED_RUNNING=0
+NESTED_STATUS=0
+
+# Create nested container fedora inside the QM.
+# Function to create and start nested container fedora, while executing podman commands from inside the QM container.
+# Extract nested container's valuable information and store it inside the global variables for later use.
+# Globals:
+#  NESTED_NAME: The nested container name in the QM container.
+#  NESTED_STATUS: The nested container status in the QM container.
+#  IS_NESTED_RUNNING: The nested container is running true or false value.
+create_nested() {
```

Comment:
There is command in common 
https://github.com/containers/qm/blob/main/tests/ffi/common/prepare.sh#L51

you can modify it yo your needs and call it,
Please avoid duplication

Reply:
@Yarboa @nsednev @pengshanyu humm I am wondering that this code is not working anymore as there is `--net host` but for sure it's a good idea from Yariv to re-use our base code. Today we use `--private` and `--host` probably **WONT** work anymore. https://github.com/containers/qm/blob/main/tests/ffi/common/prepare.sh#L62

For me, it's okay to merge @nsednev patch if it works for now to unblock him. We just need to open an issue in the github issue page as follow up (for him, my suggestion at least) so he him can keep learning from our code base. @nsednev please note, Yariv is traveling for work and he might take time for replies.

Reply:
@nsednev a good context learning here: https://github.com/containers/qm/issues/408

Reply:
yes, I think we need to update this code according to the changes in the network.